Gemfile

```diff
@@ -64,5 +64,8 @@ group :production do
   gem 'pg'
 end
 
-# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem\
+gem 'bootstrap'
```

Comment:
Do we need bootstrap? @ckjones818

Reply:
What's in this PR is all I did -- I didn't add anything else or manually install any bootstrap. The plugin seems to use some bootstrap functionality but installation was just a pretty standard add to gemfile/bundle install/add require statements process so I assume it's all internal

Reply:
@lilyosah